SCRIPTS.md

```diff
@@ -92,4 +92,4 @@
 | 88\.     | Traffic-Sign-Detection  | Using German Traffic Sign Recognition Benchmark, in this script I have used Data Augmentation and then passed the images to miniVGGNet (VGG 7) a famous architecture of Convolutional Neural Network (CNN). | [Take Me](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Traffic-Sign-Detection) |  [Vybhav Chaturvedi](https://github.com/vybhav72954) |
 | 89\.     | Voice-Assistant  | Voice-Assistant Bot | [Take Me](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Voice-Assistant) |  [Avinash .K. Rajan](https://github.com/avinashkranjan) |
 | 90\.     | Website-Status-Checker  | This script will check the status of the web address which you will input | [Take Me](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Website-Status-Checker) |  [Avinash .K. Rajan](https://github.com/avinashkranjan) |
-| 91\.     | Movie-Genre-Prediction-Chatbot       | An chatbot will predict the genre of the movie based on the input text. | [Take Me](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Movie-Genre-Prediction-Chatbot) | [Hritik Jaiswal](https://github.com/hritik5102) |
+| 91\.     | Movie-Genre-Prediction-Chatbot       | An chatbot will predict the genre of the movie based on the input text. | [Take Me](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Movie-Genre-Prediction-Chatbot) | [Hritik Jais
```

Comment:
you have modified the file, revert all the changes in this file to normal

Reply:
@kaustubhgupta  I have made the changes to the file please review